libs/stats/odc/stats/model.py

```diff
@@ -234,37 +236,63 @@ def metadata_path(self, relative_to: str = 'dataset', ext: str = 'yaml') -> str:
     def render_metadata(self, ext: str = EXT_TIFF,
                         processing_dt: Optional[datetime] = None) -> Dict[str, Any]:
         """
-        Put together EO3 metadata document for the output of this task.
+        Put together STAC metadata document for the output of this task.
         """
         if processing_dt is None:
             processing_dt = datetime.utcnow()
 
         product = self.product
         geobox = self.geobox
         region_code = product.region_code(self.tile_index)
+        inputs = list(map(str, self._lineage()))
+
         properties = deepcopy(product.properties)
 
         properties.update(self.time_range.to_stac())
         properties['odc:processing_datetime'] = format_datetime(processing_dt, timespec='seconds')
         properties['odc:region_code'] = region_code
+        properties['odc:lineage'] = dict(inputs=inputs)
+        properties['odc:product'] = product.name
+        properties['proj:epsg'] = geobox.crs.epsg
+        properties['platform'] = "sentinel-2"
+
+        assets = {
+            band: {
+                'title': band,
+                'type': "image/tiff; application=geotiff",
+                'roles': [
+                    "data"
+                ],
+                'href': path,
+                'proj:shape': geobox.shape,
+                'proj:transform': geobox.transform
+                }
+            for band, path in self.paths(ext=ext).items()
+        }
 
-        measurements = {band: {'path': path}
-                        for band, path in self.paths(ext=ext).items()}
-
-        inputs = list(map(str, self._lineage()))
+        links = [
+            {
+                "rel": "self",
+                "type": "application/json",
+                "href": self.metadata_path('absolute', ext='json')
+            },
+            {
+                "rel": "product_overview",
+                "type": "application/json",
+                "href": product.href
+            }
+        ]
+
+        geobox_wgs84_json = geobox.extent.to_crs('epsg:4326', resolution=math.inf).json
 
         return {
-            '$schema': 'https://schemas.opendatacube.org/dataset',
+            "type": "Feature",
+            "stac_version": "1.0.0-beta.2",
             'id': str(self.uuid),
-            'product': dict(name=product.name,
-                            href=product.href),
-            'location': self.metadata_path('absolute', ext='yaml'),
-
+            "bbox": [geobox_wgs84_json['coordinates'][0][0] + geobox_wgs84_json['coordinates'][0][2]],
```

Comment:
this does not make a valid bounding box, we have tools for computing bounding boxes from geometry, should use those.

Reply:
What's the tool?

Reply:
`.boundingbox` or something, like that

Reply:
you'll have to delay converting to json to use it.